spec/support/helpers/general_spec_helpers.rb

```diff
@@ -13,10 +13,21 @@ def parse_json(body)
   def expectation_file(filename)
     File.read(Rails.root.join("spec", "expectations", filename))
   end
-  
+
   def media_fixture(name)
-    path = Rails.root.join("spec/fixtures/media/#{name}")
-    File.open(path)
+    File.open(fixture("media", name))
+  end
+
+  def audio_fixture(name)
+    fixture("media", "audio", name)
+  end
+
+  def option_set_fixture(name)
+    File.open(fixture("option_set_imports", name))
+  end
+
+  def user_batch_fixture(name)
+    File.open(fixture("user_batches", name))
```

Comment:
I agree, this is not optimal since you're repeating a lot of logic. How about putting it back the way it was and using `audio_fixture(...).path` to retrieve the file's path in the test?

Reply:
cool
didn't know I could do that